### ISSUE TITLE
Proposed feature #3 - non-RDB images

### ISSUE THREAD

**Issue by roc-italy** [henrikstengaard/hst-imager#4]
This is difficult and likely not feasible/appropriate.

The UAE4All (at least the PS-Vita version) unfortunately works only with non-RDB images. You have created a HstWB version for it, so you know this well.

Now, Hst-image is meant for mastering RDB images, while a non-RDB seems really out of scope.

Nevertheless, if this could create non-RDB images (I guess the only option is FFS here) it would close another corner-case.

Once again, thank you for your consideration.

**Comment by henrikstengaard** [comment]
Non-RDB images or single partition hdf files would require geometry defined for each command. However only following commands would be of interest for Non-RDB images:
- blank
- rdb part add (only dos type will be used, size will either be ignored as its given by the image file size or the image file is resized to the given size)
- rdb part format

Non-RDB images can currently be created by exporting a partition to a file. This will function same way as a Non-RDB image and WinUAE requires sectors set to 63 instead of 32 when added (63 if default sectors used when running rdb init).

**Comment by henrikstengaard** [comment]
I worked on support for formatting non-RDB partitions and seems to got it working with latest released version v0.9.257. This has only been tested with WinUAE and not with an emulator like UAE4ALL on a PS Vita.

Here's an example of creating and formatting a DOS\3 non-RDB partition image file.
```
hst.imager.exe blank dos3_non-rdb.hdf 10mb
hst.imager.exe rdb part format dos3_non-rdb.hdf 1 WorkbenchDOS3 --non-rdb --dos-type DOS3
```

And an example of creating and formatting a PFS\3 non-RDB partition image file.
```
hst.imager.exe blank pfs3_non-rdb.hdf 10mb
hst.imager.exe rdb part format pfs3_non-rdb.hdf 1 WorkbenchPFS3 --non-rdb --dos-type PFS3
```

Adding non-RDB in WinUAE requires setting geometry surfaces to 16 and sectors 63 (others if different CHS is used) and select file sys.
![winuae_dos3_non-rdb](https://user-images.githubusercontent.com/9459668/206038875-b6723561-d552-404b-9bd3-b4e76368e777.PNG)
![winuae_pfs3_non-rdb](https://user-images.githubusercontent.com/9459668/206038978-8bd7c984-c0c4-4278-8bda-cd6992397952.PNG)

Booting WinUAE with Amiga OS 3.1 Workbench adf and non-RDB image file shows following:
![amiga-os-310-workbench_004](https://user-images.githubusercontent.com/9459668/206039725-fc347123-4cb7-447f-aef6-b9f825b2a24b.png)
![amiga-os-310-workbench_003](https://user-images.githubusercontent.com/9459668/206039791-18af881b-d894-4bf0-95ea-d3a3cffcf5c1.png)

**Comment by henrikstengaard** [comment]
I have a PS Vita and need to do some testing to see what's possible. It sounds more like it's the size of the image thats the limiting factor and not what surfaces or sectors are used. But I do recall I prepared the UAE4ALL image files with surfaces set to 2 and sectors to 32. Maybe UAE4ALL supports PFS3 by manually editing the config files, I need to test it.

I knew it would be useful to tweak cylinders, heads and sectors so the option is already available to format with surfaces set to 2 and sectors set to 32 with following commands:
```
hst.imager.exe blank dos3_non-rdb.hdf 1900mb
hst.imager.exe rdb part format dos3_non-rdb.hdf 1 WorkbenchDOS3 --non-rdb -chs 0,2,32 --dos-type DOS3
```

The trick is the option `-chs 0,2,32` when formatting, which sets cylinders to 0 (0 = max cylinders calculated), heads to 2 (same as surfaces) and sectors to 32.

The 2 commands outputs the following:
```
hst.imager.exe blank dos3_non-rdb.hdf 1900mb
[16:35:11 INF] Hst Imager v0.1.0 (12/09/2022 12:48:18)
[16:35:11 INF] Henrik Noerfjand Stengaard
[16:35:11 INF] [CMD] blank dos3_non-rdb.hdf 1900mb
[16:35:11 INF] Creating blank image at 'dos3_non-rdb.hdf'
[16:35:11 INF] Size '1.9 GB' (1992294400 bytes)
[16:35:11 INF] Done

hst.imager.exe rdb part format dos3_non-rdb.hdf 1 WorkbenchDOS3 --non-rdb -chs 0,2,32 --dos-type DOS3
[16:35:16 INF] Hst Imager v0.1.0 (12/09/2022 12:48:18)
[16:35:16 INF] Henrik Noerfjand Stengaard
[16:35:16 INF] [CMD] rdb part format dos3_non-rdb.hdf 1 WorkbenchDOS3 --non-rdb -chs 0,2,32 --dos-type DOS3
[16:35:16 INF] Formatting partition in Rigid Disk Block at 'dos3_non-rdb.hdf'
[16:35:16 INF] - Size '1.9 GB' (1992294400 bytes)
[16:35:16 INF] - Cylinders '60800'
[16:35:16 INF] - Heads '2'
[16:35:16 INF] - Sectors '32'
[16:35:16 INF] - Low Cyl '0'
[16:35:16 INF] - High Cyl '60799'
[16:35:16 INF] - Reserved '2'
[16:35:16 INF] - DOS type '0x444F5303' (DOS\3)
[16:35:16 INF] - Volume name 'WorkbenchDOS3'
[16:35:17 INF] Done
```
